llms/mlx_lm/models/plamo2.py

```diff
@@ -392,8 +400,8 @@ def __call__(
         k = k.reshape(B, T, self.k_num_heads, self.qk_dim).transpose(0, 2, 1, 3)
         v = v.reshape(B, T, self.v_num_heads, self.v_dim).transpose(0, 2, 1, 3)
 
-        q = mx.fast.layer_norm(q, None, None, 1e-6) * self.q_weight[:, None]
-        k = mx.fast.layer_norm(k, None, None, 1e-6) * self.k_weight[:, None]
+        q = _rms_norm(q, 1e-6) * self.q_weight[:, None]
+        k = _rms_norm(k, 1e-6) * self.k_weight[:, None]
```

Comment:
Thanks for fixing this. Once we have the new MLX release we can use `mx.fast.rms_norm` directly (with a `None` weight) and it should be faster.